requirements.txt

```diff
@@ -1 +1,3 @@
-pygame-ce>=2.5.0
\ No newline at end of file
+pygame-ce>=2.5.0
+pygbag==0.9.1
```

Comment:
pygbag is not a requirement unless you want to use the simulator ( and then it would need extra [aiohttp asyncio_socks_server token_utils] )

Reply:
Yeah, good point. Pygbag is not a hard requirement for the game per se, but since we are going to standardize on a web distribution for the game, I would consider it required.